dockerfiles/Dockerfile-mina-archive

```diff
@@ -31,7 +31,6 @@ RUN apt-get -y update \
         postgresql \
         postgresql-contrib \
         apt-utils \
-        procps \
```

Comment:
This should not be removed, i think either the healthchecks in kubernetes or the integration tests depend on `watch`

Reply:
it's a duplicate, there's a `procps` earlier in the list